src/test/java/core/basesyntax/service/impl/TransactionServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.impl.FruitStorageDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.service.operations.OperationHandler;
+import core.basesyntax.service.operations.impl.BalanceOperationHandler;
+import core.basesyntax.service.operations.impl.PurchaseOperationHandler;
+import core.basesyntax.service.operations.impl.ReturnOperationHandler;
+import core.basesyntax.service.operations.impl.SupplyOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Set;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class TransactionServiceImplTest {
+    private static TransactionService<List<FruitTransaction>> transactionService;
+    private static final Map<String, Integer> EXPECTED_STORAGE_DATA = Map.of("apple", 90,
+            "banana", 152);
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() {
+        FruitStorageDao fruitStorageDao = new FruitStorageDaoImpl();
+        OperationStrategy operationStrategy = new OperationStrategyImpl(getOperationHandlerMap());
+        transactionService = new TransactionServiceImpl(operationStrategy, fruitStorageDao);
+
+    }
+
+    @Test
+    public void transaction_ActualDataEqualsToExpected_ok() {
+        transactionService.transaction(getValidFruitTransactionList());
+        Set<Map.Entry<String, Integer>> actualStorageData = FruitStorage.fruitStorage.entrySet();
+        assertEquals(EXPECTED_STORAGE_DATA.entrySet(), actualStorageData);
```

Comment:
```suggestion
        assertEquals(EXPECTED_STORAGE_DATA, FruitStorage.fruitStorage);
```
can we write it this way?